cmake/FlowConfig.cmake

```diff
@@ -25,7 +25,7 @@ ExternalProject_Add(FoundationDB
 ExternalProject_Add_Step(FoundationDB server
         WORKING_DIRECTORY <SOURCE_DIR>
         EXCLUDE_FROM_MAIN 1
-        COMMAND make -j8 fdbserver fdbcli TLS_DISABLED=1)
+        COMMAND BOOSTDIR=${Boost_INCLUDE_DIRS} $(MAKE) fdbserver fdbcli TLS_DISABLED=1)
```

Comment:
How does compiling FoundationDB without TLS affect your build. Shouldn't we either compile in it? or use the same flag (TLS_DISABLED??) that your project will use to compile it in

Reply:
This target is required just for tests. Document layer doesn’t have any dependency on fdbserver otherwise. We don’t run our tests with TLS.

Although, your concern is valid for code above this patch chunk, where we build FDB libraries, which the Document Layer depends on. Even there for now we are building with TLS_DISABLED. TLS work Xin doing is going to change that. And it’s going to inherit TLS flags from parent build.

Reply:
@apkar is right. My TLS changes will have a `TLS_DISABLED` in the parent CMake file.